src/main/java/com/rpietraszewski/medicalclinic/exception/PatientEmailAlreadyExistsException.java

```diff
@@ -0,0 +1,12 @@
+package com.rpietraszewski.medicalclinic.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+@Data
+@AllArgsConstructor
+public class PatientEmailAlreadyExistsException extends RuntimeException {
```

Comment:
Zdefiniuj wyjątek MedicalClinicException, niech zawiera on pole HttpStatus status i rozszerza RuntimeException,
Natomiast kolejne wyjątki wszystkie definiowane przez nas niech rozszerzaja MedicalClinicException a nastepnie dostosuj handler oraz miejsca w ktorych rzucasz wyjatki tak aby to Service odpowiadał za to jaki komunikat zostanie zwrócony użytkownikowi.